src/EncompassRest/Token/AccessToken.cs

```diff
@@ -14,8 +14,10 @@ namespace EncompassRest.Token
 {
     public sealed class AccessToken : ApiObject
     {
-        private HttpClient _tokenClient;
-        private readonly string _apiClientId;
+        internal HttpClient _tokenClient;
+		internal TimeSpan _TimeOut = new TimeSpan(0, 0, 100);
```

Comment:
For private fields use camelCasing preceded with an underscore. Also, I'd recommend using `TimeSpan.FromSeconds(100)` instead.